docs/devGuide/development/workflow.md

```diff
@@ -126,6 +126,20 @@ The sections below has more information about various stages of submitting a PR.
 
   </modal>
 
+#### Keeping your fork up to date
+
+When new PRs are merged into the `master` branch, your `master` branch will be out of sync with the main repository. One way to update your branches to branch from the latest master is as follows:
+
+1. Go to your fork on Github and click 'Sync Fork'
```

Comment:
Would an image here showing the Sync Fork button here be good?

Reply:
yep i think thats a good idea! I will include it in the next commit